recipes/unidock/0001-disable-sm_90.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/unidock/CMakeLists.txt b/unidock/CMakeLists.txt
+index 7ceec5a..786bef7 100644
+--- a/unidock/CMakeLists.txt
++++ b/unidock/CMakeLists.txt
+@@ -23,7 +23,7 @@ if(NOT DEFINED CMAKE_CUDA_ARCHITECTURES)
+ 		80 # A100, A800
+ 		# 86 # RTX 30
+ 		# 89 # RTX 40, L40
+-		90 # H100
++		# 90 # H100
+ 	)
+ endif()
```

Comment:
Is this patch used? If not, can we drop it?

Should add that CUDA 11.8+ includes `sm_90`, which is what this recipe builds with already. So we may not need it either

```suggestion
```

Reply:
> Is this patch used? If not, can we drop it?

Will do. This patch was initially intended to bulid the package correctly under cuda 11.2 . However, some instructions used is not supported on cuda 11.2.